sky/unit/test/widget/widget_tester.dart

```diff
@@ -2,36 +2,44 @@ import 'dart:sky' as sky;
 
 import 'package:sky/animation.dart';
 import 'package:sky/rendering.dart';
-import 'package:sky/widgets.dart';
+import 'package:sky/src/fn3.dart';
 
 import '../engine/mock_events.dart';
 
-typedef Widget WidgetBuilder();
-
-class TestApp extends App {
-
-  WidgetBuilder _builder;
-  void set builder (WidgetBuilder value) {
-    setState(() {
-      _builder = value;
-    });
-  }
+class RootComponent extends StatefulComponent {
+  RootComponentState createState() => new RootComponentState();
+}
 
-  Widget build() {
-    if (_builder != null)
-      return _builder();
-    return new Container();
+class RootComponentState extends State<RootComponent> {
+  Widget _child = new DecoratedBox(decoration: new BoxDecoration());
+  Widget get child => _child;
+  void set child(Widget value) {
+    if (value != _child) {
+      setState(() {
+        _child = value;
+      });
+    }
   }
+  Widget build(BuildContext context) => child;
 }
 
 class WidgetTester {
-  WidgetTester() {
-    _app = new TestApp();
-    runApp(_app);
-    scheduler.beginFrame(0.0); // to initialise the app
+
+  // See thttps://github.com/flutter/engine/issues/1084 regarding frameTimeMs vs FakeAsync
```

Comment:
typo in comment (it was there in the previous version too, but while we're at it...)